
neo_threshold_init: 50
use_plasticity: true

use_sim_time: True